android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -210,9 +202,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
         setUpToolbar();
         setUpNavDrawer();
-        setUpProgressBar();
         setUpCustomTab();
 
+        completeHandler = DownloadCompleteHandler.with(context);
```

Comment:
Unable to understand why do this instead of just creating a new object of the class. Could learn something new.